helm/aws-operator/templates/_helpers.tpl

```diff
@@ -13,6 +13,30 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create a name stem for resource names
+
+When pods for deployments are created they have an additional 16 character
+suffix appended, e.g. "-957c9d6ff-pkzgw". Given that Kubernetes allows 63
+characters for resource names, the stem is truncated to 47 characters to leave
+room for such suffix.
+*/}}
+{{- define "resource.default.name" -}}
```

Comment:
Should we prefix with `aws-operator.` for consistency? E.g. `aws-operator.resource.default.name`.

Reply:
How about removing all `aws-operator` prefix? So we can then just copy this file over without having to rename those variables every-time.

Reply:
I'm not against that but I feel it would make Mr. @mig4 sad. He based that on upstream charts. E.g:
 https://github.com/helm/charts/blob/c1bd92189296e4e3ae248f27da4fc7c2138b2648/stable/chartmuseum/templates/_helpers.tpl#L19

Reply:
But since it would allow simply copy/pasting stuff I'm leaning towards this solution. I basically don't see any downsides other than not being compatible with upstream.

Maybe for managed apps that is a problem if they defined dependencies? But I feel that helm dependencies is an evil.

Reply:
I see no issue with not being compatible with upstream. When there's a real necessity to do this we can change this in the future, but I'd like to keep it simple and allow copy/pasting of this helpers.

Again we for helm dependency, we are no there yet and when that's an issue we can change it in the future.

Reply:
Fine with me.

Reply:
Going with no prefix, happy to help when that's an issue in the future.